src/GraphQL/Types/Collections/SchemaTypes.cs

```diff
@@ -160,7 +160,7 @@ private void CheckSealed()
             Func<Type, IGraphType> resolveType,
             ISchema schema)
         {
-            var lookup = new SchemaTypes(schema.NameConverter ?? CamelCaseNameConverter.Instance, schema.Features.AppliedDirectives);
+            var lookup = new SchemaTypes(schema.NameConverter ?? CamelCaseNameConverter.Instance, schema.Features.AppliedDirectives, schema.Features.RepeatableDirectives);
```

Comment:
Maybe it's time to pass a single `schema` argument?

Reply:
Sure. I didn’t realize you changed it back previously. Please make it require ISchema, not Schema.

Reply:
Done.